azurerm/resource_arm_key_vault_certificate.go

```diff
@@ -549,13 +615,31 @@ func flattenKeyVaultCertificatePolicy(input *keyvault.CertificatePolicy) []inter
 		for _, usage := range *props.KeyUsage {
 			usages = append(usages, string(usage))
 		}
+
+		sanOutput := make(map[string]interface{}, 0)
+		if san := props.SubjectAlternativeNames; san != nil {
+			if emails := san.Emails; emails != nil {
```

Comment:
which would mean we'd add the `map[string]interface{}` inside the if statement:

```suggestion
			sanOutput := make(map[string]interface{}, 0)
			if emails := san.Emails; emails != nil {
```